lib/ui/platform_dispatcher.dart

```diff
@@ -1314,6 +1371,100 @@ class WindowPadding {
   }
 }
 
+/// Area of the display that may be obstructed by a hardware feature.This is
+/// populated only on Android.
+///
+/// The [bounds] are measured in logical pixels. On devices with two screens the
+/// coordinate system starts with [0,0] in the top-left corner of the left screen
+/// and expands to include the visual space between the screens and the right screen.
+/// The [type] can be used to determine if this display feature obstructs the screen or not.
```

Comment:
New paragraph for new concept.

Reply:
Also, this can probably be better worded, the primary purpose of type doesn't seem to be whether it obstructs or not, rather it seems to express the shape and nature of the feature.